WordPress/gutenberg
# Remove fill="none" from levelUp icon

## What?
This removes `fill="none"` from this SVG.

## Why?

There's a little bug in this icon file making the icon invisible when you import it in. Removing `fill="none"` fixes the issues and makes it the same as other icons.

## Testing Instructions




1. import `levelUp` from the icons library without this PR, and see that the icon is invisible
2. import `levelUp` from the icons library with this PR, and see that the icon now shows up.

I'm not sure if there is some build scripts or anything I have to run in order for this PR to be successful, if someone could let me know, that'd be helpful!
